### DISCUSSION TITLE
[@types/express-session] What is the `secure` property of CookieOption?

### DISCUSSION THREAD

**Discussion by BOKYUNGGG** [DefinitelyTyped/DefinitelyTyped#68476] [Issues with a @types package] [answered]
Hello,

I hope this message finds you well. I have a question regarding the express-session library, specifically version 1.18.0, and its TypeScript definitions in DefinitelyTyped. I have noticed that the `signed` property of `CookieOption` is not valid in the library (version 1.18.0). While the type definition is present in DefinitelyTyped, there is no accompanying documentation in the comments. Additionally, I couldn't find any information regarding the `signed` attribute in the Set-Cookie HTTP Response Header as well.
```typescript
interface CookieOptions {
   //...
   signed?: boolean | undefined;

}
```

My questions are as follows:

1. What is the `signed` property in CookieOption as specified in @types/express-session?
2. If there is misinformation or a lack of documentation, what steps should I take to correct this?

Thank you for your time and assistance.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
Hey, the change that introduced the property is from 2020, afaik that one:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46576/files#diff-c38f30a0fdd238f104a7392ff3881fa97d8b4497d75e9617163ac1b5fceb75bf
Some properties within this DT package are just access properties to properties of underlying type from other package, here from `cookie`, if I'm not wrong.
It could be that implementation derailed from current state, became outdated, etc.
Current object info:
https://github.com/jshttp/cookie/blob/38323bad3aa04bce840103ff6075bc05cc0bf884/index.js#L101
usually PR with update, that's the best solution, thanks!
